processor/labelfilterprocessor/config.go

```diff
@@ -0,0 +1,24 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package labelfilterprocessor
+
+import (
+	"go.opentelemetry.io/collector/config/configmodels"
+)
+
+type Config struct {
+	configmodels.ProcessorSettings `mapstructure:",squash"`
+	Exclude                        []string `mapstructure:"exclude"`
```

Comment:
If you use an expression, you don't really need the notion of exclude/include, just have the return value be true (include) or false (exclude).  The only reason there is the split between exclude/include in other filters is because of the limited logic available with defining the filter in yaml.

Reply:
Yeah, that does make things cleaner. Will update.